cloud-function/src/env.ts

```diff
@@ -83,6 +83,9 @@ export const BSA_ZONE_KEYS = Object.fromEntries(
 export const BSA_ENABLED = Boolean(
   JSON.parse(process.env["BSA_ENABLED"] || "false")
 );
+export const WILDCARD_ENABLED = Boolean(
```

Comment:
I would welcome a small comment here, since the name does not immediately describe its purpose